core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockIndex.java

```diff
@@ -895,7 +895,7 @@ public String getTableTaskInfo(int index) {
     return minMax;
   }
 
-  private boolean[] getMinMaxFlag(IndexRow row, int index) {
+  public static boolean[] getMinMaxFlag(IndexRow row, int index) {
```

Comment:
Move it to a utility class

Reply:
removed this change, as it was not required

Reply:
reverted this as not required